ecs-agent/daemonimages/credentialsfetcher/Dockerfile

```diff
@@ -14,17 +14,15 @@
 # use minimal amazonlinux:2023 base for credentials-fetcher image
 FROM public.ecr.aws/amazonlinux/amazonlinux:2023-minimal
 
-RUN dnf install -y krb5-workstation jq unzip && mkdir /krb5 && chmod 755 /krb5
+RUN dnf install -y krb5-workstation
 RUN dnf install -y sssd
-RUN dnf install -y realmd
```

Comment:
I'm assuming `RUN dnf install -y krb5-workstation` replaces all the previous commands associated with krb5 that are being removed as part of this pull request (please correct me if my assumption is wrong).

Curious why we are also removing the install realmd command here. Is it also included as part of krb5-workstation?

Reply:
only realmd is removed

Reply:
Synced offline, Sai confirmed that realmd is not needed for the use-case.